recipes/xsum/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "xsum" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21ff547dbb64e8ddefd376f2889349c7a94fce69e7fc417fc661068b298473b1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
You porbably can't be noarch-python and rely on a C++ compiler

Reply:
@scopatz Thanks for your answer, but I am not sure that is the reason. I looked at some other recipes that are suing C/C++ compilers, and they are noarch-python. Maybe it is wrong, but they encounter no problem.

Reply:
@scopatz Thank you very much. That was the issue. This package is not a pure-Python package, and I forgot to remove the noarch.